dashboard/templates/dashboard/home.html

```diff
@@ -6,7 +6,7 @@
 {% block content %}
 <div class="row">
     <div class="large-6 columns">
-        <h2>{% trans "Monitor your apps in Firefox" %}</h2>
+        <h1>{% trans "Monitor your apps in Firefox" %}</h1>
```

Comment:
Nice. We should definitely have an `<h1>` on the page.

I would even recommend that we make the main title in the heading an `<h1>` and keep this page title as an `<h2>`. That's what we do on [mozilla.org](https://www.mozilla.org/), [Hacks](https://hacks.mozilla.org/), and elsewhere.